addon/helpers/format-number.js

```diff
@@ -5,21 +5,35 @@ function isDecimalNumber(value) {
   return value % 1 !== 0;
 }
 
-export function formatNumber(value = 0, decimalPlaces = 0) {
+export function formatNumber(value = 0, precision = 0) {
   value = unformat(value + '');
   const absoluteValue = Math.abs(value);
   let formattedValue = absoluteValue + '';
-  if (decimalPlaces !== undefined || isDecimalNumber(value)) {
-    formattedValue = absoluteValue.toFixed(decimalPlaces);
+  let decimals = '';
+  if (precision !== undefined || isDecimalNumber(value)) {
+    formattedValue = absoluteValue.toFixed(precision);
+    let splits = formattedValue.split('.');
+
+    if (splits[0] != undefined) {
+      formattedValue = splits[0];
+    }
+
+    if (splits[1] != undefined) {
+      decimals = `.${splits[1]}`;
+    }
```

Comment:
Calling `splits[1] != undefined` can still throw an array out of bounds exception here.
```suggestion
  if (precision !== undefined || isDecimalNumber(value)) {
    formattedValue = absoluteValue.toFixed(precision);
    const splits = formattedValue.split('.');
    if(splits.length == 2){
      formattedValue = splits[0];
      decimals = `.${splits[1]}`;
    }
```